packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -251,7 +251,8 @@ const main = async () => {
             execSync(`${babelNode} ${inspect ? '--inspect' : ''} ${entrypoint}`, {
                 env: {
                     ...process.env,
-                    ...(noHMR ? {HMR: 'false'} : {})
+                    ...(noHMR ? {HMR: 'false'} : {}),
+                    ...{WEBPACK_INSPECT: inspect}
```

Comment:
```suggestion
                    ...(noHMR ? {HMR: 'false'} : {})
```
A separate environment variable is not needed, as the app can detect `--inspect` on its own.